sidebars.js

```diff
@@ -976,6 +976,12 @@ module.exports = {
     {
       "Zeebe Gateway": ["self-managed/zeebe-gateway-deployment/zeebe-gateway"],
     },
+    {
+      "Backup & Restore": [
+        "self-managed/backup-restore/backup-and-restore",
+        "self-managed/backup-restore/zeebe-backup-and-restore",
+      ],
+    },
```

Comment:
Due to the fact that we now have separate versions for Optimize than the rest of the docs, parallel sidebar changes will need to be made at https://github.com/camunda/camunda-platform-docs/blob/main/optimize_sidebars.js#L1540-L1541.